File: 7. Reverse a string/code.cpp
#include <iostream>
#include <algorithm>
using namespace std;

void reverseString(string& s) {
    reverse(s.begin(), s.end());
}

int main() {
    string input_str;
    cout << "Enter a string: ";
    getline(cin, input_str);
    reverseString(input_str);
    cout << "Reversed string: " << input_str << endl;
    return 0;
}

File: 7. Reverse a string/code.java
import java.util.Scanner;

public class Main {
    public static String reverseString(String s) {
        return new StringBuilder(s).reverse().toString();
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a string: ");
        String inputStr = scanner.nextLine();
        System.out.println("Reversed string: " + reverseString(inputStr));
    }
}
